### ISSUE TITLE
Create a Heroku build

### ISSUE THREAD

**Issue by viktorsmari** [piratar/wasa2il#97]
Would be nice if someone could make wasa2il build on Heroku.

Then we could push our staging/test branch to a Heroku remote to show/test our new feature.

**Comment by BjarniRunar** [comment]
Never used Heroku, no opinion there. However, I'm happy to give away sponsored https://pagekite.net accounts for people who want to show and get feedback on wasa2il works-in-progress. Interested parties should sign up and then send me an e-mail mentioning wasa2il and their account.

**Comment by BjarniRunar** [comment]
(sorry about the dup, I posted this on a PR when it probably belonged here)

If this is a request for a staging server, I'd like to say: I'm not a fan of staging servers. They're just another thing to admin, with questionable benefits.

The friction we have now has the positive side-effect of keeping the delta between what is live and running, and what the devs are hacking on small. People are wary of making large breaking changes, which is mostly a good thing. Projects with a nice separate dev environment tend to diverge more radically from production, often leading to a vicious cycle of production upgrades becoming more painful, and thus avoided, and thus diverging further and becoming more painful... rather than work on a staging server, I spent time on creating a set of test data to make local testing easier.

Obviously I'm not going to prevent others from setting up a staging server, but I won't be spending time on it myself. :-)
